engine/src/w32-clipboard.cpp

```diff
@@ -1246,6 +1309,10 @@ MCDataRef MCWin32RawClipboardItemRep::CopyData() const
     }
     
 	// Update the data cache and return it
+	if (t_data == nullptr)
```

Comment:
@livecodealex hmm... still confused why this resolves the crash... I'm wondering if it's an issue with the auto value. Perhaps try using this which seems to handle the case of the value being assigned being null:
```
m_bytes.Reset(t_data)
```
Actually I'm not really sure why the `=` operator doesn't behave like `Reset`.